layouts/partials/handle-version-switch-404.html

```diff
@@ -0,0 +1,11 @@
+{{/*
+If a 404 occurs and the referrer is a different version of the Kubernetes documentation,
+don't show the regular 404 error page, but instead redirect to the site root
+*/}}
+<script type="text/javascript">
+  var sameBaseUrlRegExp = new RegExp("^" + location.protocol + "//" + location.host.replace(".", "\\.") + "/.*", "i");
+  var kubernetesDocumentationUrlRegExp = new RegExp("^https?://(.+\\.)?kubernetes.io/.*", "i");
```

Comment:
```suggestion
  var kubernetesDocumentationUrlRegExp = new RegExp("^https?://(.+\\.)?{{ $.Site.BaseURL }}/.*", "i");
```
?

Reply:
I take it back - the original is correct and the replacement is wrong.

Reply:
(sorry)

Reply:
Its ok, it looked like a good idea to me